rsrcdump/pict.py

```diff
@@ -112,9 +118,8 @@ def unpackbw(u: Unpacker, bm: Bitmap):
                 dst.write(b'\xFF\xFF\xFF\xFF')
     return dst.getvalue()
 
-def unpack0(u: Unpacker, pmh: Pixmap, palette): # w, h, rowbytes, palette):
-    unpacked = unpack_all_rows(u, ">B", numrows=pmh.frame_h, rowbytes=pmh.rowbytes)
-    unpacked = bytes(unpacked)
+def unpack0(u: Unpacker, pmh: Pixmap, palette: List[bytes]) -> bytes: # w: int, h: int, rowbytes: bytes, palette: List[bytes]) -> bytes:
```

Comment:
Let's get rid of this obsolete comment while we're here.
```suggestion
def unpack0(u: Unpacker, pmh: Pixmap, palette: list[bytes]) -> bytes:
```